src/js/edu-benefits/5495/config/form.js

```diff
@@ -43,7 +43,7 @@ const formConfig = {
   trackingPrefix: 'edu-5495-',
   formId: '22-5495',
   version: 0,
-  disableSave: true,
+  // disableSave: true,
```

Comment:
I had also added in `savedFormMessages`, based on what we did for pensions and hca.